cmd/config/config.go

```
@@ -282,233 +271,93 @@ func (s *syncCommand) setDefaultApplicationPath() (string, error) {
+	}
 	if s.host != "" {
 		return nil
 	}
-	storedAuth, err := loadStoredTokens()
-	if err != nil {
-		return err
-	}
-	if storedAuth.Token == "" {
-		return fmt.Errorf("stored authentication is invalid. Please login")
+	if _, err := loadStoredAuth(); err != nil {
+		return fmt.Errorf("Hub authentication required for sync: %w", err)
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)